mw-webapp/src/component/select/Select.cy.tsx

```diff
@@ -65,16 +62,18 @@ describe("Select component", () => {
   });
 
   it("should change value inside trigger if some selectItem selected", () => {
-    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(SELECT_OPTIONS[FIRST_OPTION_INDEX].text);
+    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(
+      SELECT_OPTIONS[FIRST_OPTION_INDEX].text,
+    );
     cy.get(getDataCy(SELECT_CY.dataCyTrigger)).click();
-    cy.get(getDataCy(SELECT_CY.dataCyContentList)).click();
-    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(SELECT_OPTIONS[SECOND_OPTION_INDEX].text);
+    // Force required for correct work for MacOS
+    cy.get(getDataCy(SELECT_CY.dataCyContentList)).click({force: true});
```

Comment:
Can you please tell more about why on MacOs doesn't work click()? Or send link please on resource

Reply:
I received this error!
```
Select component
should change value inside trigger if some selectItem selected:
AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Select text 2' within the element: ‹span> but never did.
```